### PR TITLE
Added the base functionality

### PR DESCRIPTION
**Pull request by kirovboris**
/cc @DevExpress/testcafe-docs
Please review, especially with repo structure, publish options etc.

### CODE REVIEW

**Comment by kirovboris** [comment]
README pr is coming soon

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "testcafe-react-plugin",
```
Comment (inikulin) [inline package.json]: It should be `testcafe-react-selectors`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "version": "0.0.1",
+  "description": "TestCafe plugin for the testing ReactJS Components",
```
Comment (inikulin) [inline package.json]: It's React selectors for TestCafe

**Comment by kirovboris** [comment]
FPR

**Comment by kirovboris** [comment]
FPR

**Comment by kirovboris** [comment]
@VasilyStrelyaev please check the strings

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+export default Selector(selector => {
+    const SUPPORTED_REACT_VERSION = 15;
+
+    if (!window.React)
+        return document.querySelectorAll(selector);
+
+    const reactVersion = parseInt(window.React.version.split('.')[0], 10);
+
+    if (reactVersion < SUPPORTED_REACT_VERSION)
+        throw new Error('testcafe-react-selectors supports React js starting with 15.x version');
```
Comment (VasilyStrelyaev) [inline src/index.js]:
```
        throw new Error('testcafe-react-selectors supports React version 15.x and newer');
```

File: test/react-plugin-test.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+    try {
+        await ReactSelector('App');
+    }
+    catch (e) {
+        await t.expect(e.errMsg).contains('testcafe-react-selectors supports React js starting with 15.x version');
+    }
+});
+
+test('There is not React js on the testing page', async t => {
```
Comment (VasilyStrelyaev) [inline test/react-plugin-test.js]:
```
test('There is no React on the tested page', async t => {
```

**Pull request merged**